Code/RDBoost/Wrap.h

```diff
@@ -18,6 +18,10 @@
 //
 #include <boost/python.hpp>
 #include <boost/python/stl_iterator.hpp>
+#include <memory>
+// ha ha... we don't actually want to use this, but we need to keep support old
+// C++ versions:
+#define rdk_unique_ptr std::auto_ptr
```

Comment:
Changing the name to rdk_auto_ptr to avoid confusion is probably a good idea.
I'll update the PR to do that and include a C++11 compatible version.